src/celeritas/user/detail/SimpleCaloImpl.hh

```diff
@@ -0,0 +1,37 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file celeritas/user/detail/SimpleCaloImpl.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include "corecel/Macros.hh"
+#include "corecel/Types.hh"
+
+#include "../SimpleCaloData.hh"
```

Comment:
We've always used full paths in our includes. Not a big deal as a detail implementation, but easy to break if we decide to move things around.

Reply:
You're right! I didn't remember we'd use that and should have given a second look before commenting.